CIP-0062/README.md

```diff
@@ -0,0 +1,530 @@
+data. This CIP will eliminate that requirement and give these systems a clear
+and unambiguous way to signal to the wallet that the data that requires signing
+is a Vote.
+
+# Specification
+
+## Version
+
+The API extension specified in this document will count as version `0.2.1` for
+version-checking purposes below.
+
+## Data Types
+
+### PublicKey
+
+TODO: Define this.
+
+### GovernanceKey
+
+```js
+type GovernanceKey = {
+  votingKey: string
+  weight: number
+}
+```
+
+* votingKey - A hex-encoded string representing a 32 byte Ed25519 public key as described in [CIP-36](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0036#delegation-format).
+* weight - Used to calculate the proportion of voting power, using the rules described
+  in [CIP-36](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0036#delegation-format).
+
+### VotingPurpose
+
+```ts
+type enum VotingPurpose = {
+  CATALYST = 0
+}
+```
+
+* VotingPurpose - Defines the voting purpose of the governance functions. This is used
+to limit the scope of the governance functions. For example, a voting purpose
+might denote a council election, or some private purpose (agreed by convention). Currently, only voting purpose 0 (zero) is defined, and it is for Catalyst.
+
+**IMPORTANT**: The list of purposes in this specification should be considered the
+authoritative list of known purposes, subject to future amendment. Other voting
+purposes will be defined as required, by either an update to this CIP or a
+future CIP listing currently allocated voting purposes.
+
+### BlockDate
+
+```ts
+interface BlockDate {
+  epoch: number
+  slot: number
+}
+```
+
+* epoch - an epoch value on the voting blockchain.
+* slot - a block slot number on the voting blockchain.
+
+### Proposal
+
+```ts
+interface Proposal {
+  votePlanId: string;
+  voteOptions: number[] | string[];
+  votePublic: boolean;
+  proposalIndex?: number;
+}
+```
+
+A voting proposal.
+
+Broadly, a Voting Proposal is a single item that can be voted on. The votePlanId
+groups these proposals together. How vote plans and proposals relate, how many
+there can be, and how many a voter may vote on are controlled by the voting
+dApp.
+
+* votePlanId - string representing the vote plan unique identifier. The format
+  of this string is voting system specific and not defined in this CIP.
+* voteOptions - List of possible "choices" which can be made for this proposal.
+* votePublic - Boolean indicating whether the vote is public or private.
+* proposalIndex - Optional, The index within the vote plan for this Proposal.
+
+The unique key for identifying a Proposal is a combination of `votePlanId` and
+`proposalIndex`. The dApp for the voting system is free to utilize these fields
+in any way required to support that voting system. If the voting system only
+requires a `votePlanId` the proposalIndex may be omitted.
+
+### Vote
+
+```ts
+interface Vote {
+  proposal: Proposal;
```

Comment:
```suggestion
  proposal: Proposal
```
Bruno told be that , or ; are not required in interface definitions - I had removed them throughout